<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Articles</class>
 <widget class="QDialog" name="Articles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Articles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="4">
    <widget class="QTableView" name="tableView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="volume">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QComboBox" name="vComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="issue">
       <property name="text">
        <string>Issue:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QComboBox" name="iComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="month">
       <property name="text">
        <string>Month: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="year">
       <property name="text">
        <string>Year: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Journal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="viewArticleButton">
       <property name="text">
        <string>View Article</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editArticleButton">
       <property name="text">
        <string>Edit Article</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delArticleButton">
       <property name="text">
        <string>Delete Article</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
